src/aes_key.c

```diff
@@ -1196,7 +1197,13 @@ int aes_key_clr2prot(struct zpc_aes_key *aes_key, const unsigned char *key,
 	io.pkeylen = sizeof(aes_key->prot.protkey);
 	io.pkey = (unsigned char *)&aes_key->prot.protkey;
 
-	rc = ioctl(pkeyfd, PKEY_KBLOB2PROTK3, &io);
+	for (i = 0; i < 10; i++) {
+		rc = ioctl(pkeyfd, PKEY_KBLOB2PROTK3, &io);
```

Comment:
Actually this busy situation doe sonly happen when a secure key (CCA or EP11) is transformed into a protected key. It does NOT happen for clear key tokens or other non-securekey tokens. 
Of course it does not hurt to implement the retry loop for this case as well.

Reply:
Ah ok, removed loop for clr2prot.